src/util.h

```diff
@@ -69,15 +70,50 @@ inline std::string _(const char* psz)
 void SetupEnvironment();
 bool SetupNetworking();
 
+enum LogFlags : uint32_t {
+    LOG_NONE        = 0,
+    LOG_NET         = (1 <<  0),
```

Comment:
Oh darn, didn't see this until I'd already done the rebase. Uh, I'll look into it and put it as a commit on top for squashing.  (presumably it'll be a more or less trivial change that could be reviewed separately)